worklist/helperFunctions/fetch_articles_from_petscan.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+import logging
+
+logger = logging.getLogger('django')
+
+
+# Function to fetch articles from
+# Petscan by taking a petscan ID as input
+def fetch_articles_from_petscan(psid):
+    message = 'Successfully fetched articles from petscan ' \
+              'using ID: {0}'.format(str(psid))
+    success = True
+    data = {}
+    parameters = {'format': 'json', 'psid': psid}
+    url = "https://petscan.wmflabs.org"
+    response_object = requests.get(url, params=parameters)
+    if response_object.status_code == 200:
```

Comment:
`requests` can already parse json for us, no need to do it ourselves. Also, it's a little better to assume success once we actually parse the result rather than based on the status code I think. For example, http://petscan.wmflabs.org/?psid=-1&format=json returns 200 just fine, but that's because PetScan redirected us to the main page rather than because we actually got a valid JSON response.

So, something like:

```python
try:
  response = requests.get(url, params={'format': 'json', 'psid': psid}).json()
except Exception as e:
  return False, 'Failed to query PetScan: {0} {1}'.format(type(e), e)
if 'error' in response:
  return False, ...
articles = response['*'][0]['a']['*']
...
```

Reply:
Ah, I didn't notice it earlier. Thanks. I've done the changes.